libraries/mediaupload/impl/src/main/kotlin/io/element/android/libraries/mediaupload/impl/AndroidMediaPreProcessor.kt

```diff
@@ -207,12 +207,44 @@ class AndroidMediaPreProcessor @Inject constructor(
 
     private fun removeSensitiveImageMetadata(file: File) {
         // Remove GPS info, user comments and subject location tags
-        val exifInterface = ExifInterface(file)
-        // See ExifInterface.TAG_GPS_INFO_IFD_POINTER
-        exifInterface.setAttribute("GPSInfoIFDPointer", null)
-        exifInterface.setAttribute(ExifInterface.TAG_USER_COMMENT, null)
-        exifInterface.setAttribute(ExifInterface.TAG_SUBJECT_LOCATION, null)
```

Comment:
I think it was added by mistake. The docs say:

```
Indicates the location of the main subject in the scene. The value of this tag represents
the pixel at the center of the main subject relative to the left edge, prior to rotation
processing as per {@link #TAG_ORIENTATION}. The first value indicates the X column number
and second indicates the Y row number. When a camera records the main subject location,
it is recommended that {@link #TAG_SUBJECT_AREA} be used instead of this tag.
```

So it's not user location, but some metadata to help image viewers locate the 'important' part of the image.

Reply:
OK, thanks for explaining!